livekit-agents/livekit/agents/llm/llm.py

```diff
@@ -29,6 +29,10 @@ class ChatChunk:
 
 
 class LLM(abc.ABC):
+    def __init__(self, *, temperature: float | None = None):
+        # set default temperature, used by `chat` method when temperature isn't provided
+        self._temperature = temperature
```

Comment:
If possible I would have created a private dataclass called `_LLMOptions`. But feel free to ignore since we only expose temperature for now

Reply:
hmm that's true.. we already use LLMOptions for model and user.. I'll add to that

Reply:
tho the current design they are private to OpenAI/Anthropic.